### PR TITLE
chore(readme): organize structure and content

### PR DESCRIPTION
**Pull request by nijikokun**
- Added installation with yarn and pnpm
- Moved documentation below quick start
- Moved example documentation under documentation section
- Moved builders, async and tls under documentation section
- Changed introduction
- Changed quick start
- Simplified examples list
- Simplified license section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,58 +14,73 @@
...
+`ngrok-nodejs` is the official Node.js SDK for ngrok that requires no binaries. Quickly enable secure production-ready connectivity to your applications and services directly from your code.
 
-[ngrok](https://ngrok.com) is a globally distributed reverse proxy commonly used for quickly getting a public URL to a
-service running inside a private network, such as on your local laptop. The ngrok agent is usually
-deployed inside a private network and is used to communicate with the ngrok cloud service.
-
-This is the ngrok agent in library form, suitable for integrating directly into NodeJS
-applications. This allows you to quickly build ngrok into your application with no separate process
+[ngrok](https://ngrok.com) is a globally distributed gateway that provides secure connectivity for applications and services running in any environment.
 to manage.
```
Comment (bobzilladev) [inline README.md:20]: Can remove dangling `to manage.`

File: README.md [hunk 2]
```diff
@@ -14,58 +14,73 @@
 [ci-badge]: https://github.com/ngrok/ngrok-nodejs/actions/workflows/ci.yml/badge.svg
 [ci-url]: https://github.com/ngrok/ngrok-nodejs/actions/workflows/ci.yml
 
-**Note: This is beta-quality software. Interfaces may change without warning.**
+`ngrok-nodejs` is the official Node.js SDK for ngrok that requires no binaries. Quickly enable secure production-ready connectivity to your applications and services directly from your code.
```
Comment (bobzilladev) [inline README.md:17]: Maybe `separate executable binary`

File: README.md [hunk 3]
```diff
@@ -14,58 +14,73 @@
...
+
+```shell
+pnpm add @ngrok/ngrok
 ```
 
-You can find more examples in [the /examples directory](https://github.com/ngrok/ngrok-nodejs/tree/main/examples).
+# Quickstart
+
+1. [Install `@ngrok/ngrok`](#installation)
+2. Export your [authtoken from the ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken) in your terminal
```
Comment (bobzilladev) [inline README.md:45]: Maybe mentioning `NGROK_AUTHTOKEN`

**Review by bobzilladev — APPROVED** [review]
Looks nice! Left a few comments

File: README.md (hunk 2 above)
Comment (nijikokun) [reply README.md:17]: Thought about this, but ultimately decided against it. It felt like it implied there was still a binary bundled.

File: README.md (hunk 3 above)
Comment (nijikokun) [reply README.md:45]: Good call. Should have checked, was hoping this page mentioned the actual environment variable.

Comment (bobzilladev) [reply README.md]: The env var is down in the `Authorization` section, but I do like it here for folks that glance at the quickstart and nothing else.

**Pull request merged**